doc/quickstart/run.rst

```diff
@@ -49,6 +49,40 @@ Possible values are:
   - `default`: fail if there are any errors
   - `strict`: fail if there are any warnings
 
+To re-use pre-processed files from a previous run of the same recipe, you can
+use
+
+.. code:: bash
+
+    esmvaltool run recipe_python.yml --resume_from ~/esmvaltool_output/recipe_python_20210930_123907
+
+Multiple directories can be specified for re-use, make sure to quote them:
+
+.. code:: bash
+
+    esmvaltool run recipe_python.yml --resume_from "~/esmvaltool_output/recipe_python_20210930_101007 ~/esmvaltool_output/recipe_python_20210930_123907"
```

Comment:
maybe add "and don't separate them by comma"?

Reply:
Listing all the things people shouldn't do would get a bit exhaustive ;-) Actually, `fire` allows you to give a proper Python list, but that's a bit more tricky to get right with all the quotes:
```
esmvaltool run recipe_python.yml --resume_from '["~/esmvaltool_output/recipe_python_20210930_101007", "~/esmvaltool_output/recipe_python_20210930_123907"]'
```
that's why I added this in the same way as is done for providing the `--diagnostics` you want to run.

Reply:
hahah, all cool man, hopefully they'll figure it out :grin: